### PR TITLE
cleanup PO

### LINKED ISSUE #1172: PO Cleanup post Migrate
<This is intended as a guide when thinking about or creating issues for go-centrifuge>

#### Testing
<Describe required tests: Unit, Integration, Testworld or Functional>

#### Acceptance Criteria
<What is the acceptance criteria of this ticket?>

#### Dependencies and affected external components
<Outline any dependencies and components that could be affected by this>

### PR DESCRIPTION
**Pull request by vedhavyas**
Closes #1172

### CODE REVIEW

File: documents/purchaseorder/test_purchaseorder.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+	}
+	_, err = po.CalculateDocumentRoot()
+	assert.NoError(t, err)
+	cd, err := po.PackCoreDocument()
+	assert.NoError(t, err)
+
+	return po, cd
+}
+
+func CreatePOWithEmbedCD(t *testing.T, ctx context.Context, did identity.DID, collaborators []identity.DID) (*PurchaseOrder, coredocumentpb.CoreDocument) {
```
Comment (vedhavyas) [inline documents/purchaseorder/test_purchaseorder.go]: @charleenfei this will be used across for testing

File: p2p/client_integration_test.go [hunk 2]
```diff
@@ -133,15 +131,15 @@ func prepareDocumentForP2PHandler(t *testing.T, collaborators [][]byte) document
...
 	assert.NoError(t, err)
-	payalod := testingdocuments.CreatePOPayload()
-	var cs []string
-	for _, c := range collaborators {
-		cs = append(cs, hexutil.Encode(c))
-	}
-	payalod.WriteAccess = cs
-	po := new(purchaseorder.PurchaseOrder)
-	err = po.InitPurchaseOrderInput(payalod, defaultDID)
+	payalod := purchaseorder.CreatePOPayload(t, nil)
```
Comment (mikiquantum) [inline p2p/client_integration_test.go]:
```suggestion
	payload := purchaseorder.CreatePOPayload(t, nil)
```

File: documents/purchaseorder/test_purchaseorder.go [hunk 3]
```diff
@@ -0,0 +1,110 @@
+// +build integration unit
```
Comment (mikiquantum) [inline documents/purchaseorder/test_purchaseorder.go]: I believe testworld uses it as well.

Comment (vedhavyas) [reply documents/purchaseorder/test_purchaseorder.go]: made those chnages :-D

**Pull request merged**